core/src/main/java/tachyon/worker/TachyonWorker.java

```diff
@@ -144,8 +151,11 @@ public static void main(String[] args) throws UnknownHostException {
           + "tachyon.Worker [<MasterHost:Port>]");
       System.exit(-1);
     }
-    
+
     TachyonConf tachyonConf = new TachyonConf();
+    if (args.length == 1) {
+      getMasterLocation(args[0], tachyonConf);
```

Comment:
+1 for the suggestion.

The method has side effect which is not good practice.